### ISSUE TITLE
Moderations endpoint missing from OpenAPI Specification

### ISSUE THREAD

**Issue by om4james** [openai/openai-openapi#3]
Hi there,

*Moderation* is mentioned in the docs:

- https://beta.openai.com/docs/guides/moderation/overview
- https://beta.openai.com/docs/api-reference/moderations

However the `post https://api.openai.com/v1/moderations` endpoint is not included in https://github.com/openai/openai-openapi/blob/master/openapi.yaml

It looks like this functionality was added to the [openai cli v0.20.0](https://github.com/openai/openai-python/releases/tag/v0.20.0), but done manually. Moderation doesn't seem to be included in in the [openai-node](https://github.com/openai/openai-node) as of yet.

Are there plans to add this new endpoint to the OpenAPI specification?

Adding it would presumably make it simple to add it to your node package also.

Thank you!

**Comment by om4james** [comment]
Hi there @jeevnayak and @schnerd, would either of you be able to assist with this please?

If I was to submit a pull request that adds the `moderations` endpoint to your OpenAPI specification, would that help move things forward?

Your `openai-node` package doesn't support moderations either (https://github.com/openai/openai-node/issues/19), so adding it to the spec would help with that too.

Thank you for your time 😀

**Comment by aliyeysides** [comment]
@om4james +1 from me but I wouldn't wait on it. Probably best to get around this by wrapping the openai node client and making the call to the `moderations` endpoint yourself. When they eventually get to this, you can just change the implementation to call the SDK method instead. 

```js
export class OpenAIClient {
  #config;
  #client;

  constructor(config) {
    if (!config || !config.apiKey)
      throw new Error(
        "Missing OpenAI configuration. Please provide an apiKey."
      );
    this.#config = config;
    this.#client = new OpenAIApi(config);
  }
  
  async moderateContent(input) {
    try {
      const response = await fetch("https://api.openai.com/v1/moderations", {
        method: "POST",
        headers: {
          "Content-Type": "application/json",
          Authorization: `Bearer ${this.#config.apiKey}`,
        },
        body: JSON.stringify({ input }),
      });
      return response.json();
    } catch (e) {
      throw e;
    }
   }
  
}

```

**Comment by schnerd** [comment]
Hey – thanks for raising this and sorry it took us so long to respond!

Thanks for offering to open a PR – we do already have a nice process in place to auto-generate the SDKs, looks like we just haven't gone through those steps yet. I'll try to do that by the end of this week. In the meantime @aliyeysides code would also work.

**Comment by schnerd** [comment]
Just added this – published in `openai@3.0.1` on npm

**Comment by om4james** [comment]
Thank you very much @schnerd for adding the missing moderations endpoint 💯
